charts/newrelic-infrastructure/README.md

```diff
@@ -1,6 +1,6 @@
 # newrelic-infrastructure
 
-![Version: 3.7.0](https://img.shields.io/badge/Version-3.7.0-informational?style=flat-square) ![AppVersion: 3.4.0](https://img.shields.io/badge/AppVersion-3.4.0-informational?style=flat-square)
```

Comment:
We are removing this from here: https://github.com/newrelic/nri-kubernetes/blob/f09ffb8/charts/newrelic-infrastructure/README.md.gotmpl#L4

Reply:
Done: [cf42430](https://github.com/newrelic/nri-kubernetes/pull/515/commits/cf42430716f61dec36385fcc0f3429a5b776ab3d)